src/transformers/utils/import_utils.py

```diff
@@ -42,6 +42,8 @@
 USE_TORCH = os.environ.get("USE_TORCH", "AUTO").upper()
 USE_JAX = os.environ.get("USE_FLAX", "AUTO").upper()
 
+FORCE_TF_AVAILABLE = os.environ.get("USE_TF", "AUTO").upper()
```

Comment:
should be FORCE_TF_AVAILABLE

Reply:
Agreed with @kevint324, I'd put this is a `FORCE_TF_AVAILABLE` env var :)

Reply:
Agh, typo, I'm sorry! That's definitely more than a nit!